linux/Dockerfile

```diff
@@ -1,90 +0,0 @@
-
-ARG EMSCRIPTEN_VERSION=3.1.44
-# Install emscripten
-RUN cd /usr/share && \
-  git clone https://github.com/emscripten-core/emsdk.git && \
-  cd emsdk && \
-  ./emsdk install ${EMSCRIPTEN_VERSION} && \
-  ./emsdk activate ${EMSCRIPTEN_VERSION} && \
-  echo "#!/bin/bash" > /usr/bin/bashwrapper && \
-  echo "export EMSCRIPTEN_BASE=/usr/share/emsdk/upstream/emscripten" >> /usr/bin/bashwrapper
-
-# Keyman Web
-RUN curl --output google-chrome-stable_current_amd64.deb https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb && \
-  apt-get -q -y install ./google-chrome-stable_current_amd64.deb && \
-  rm google-chrome-stable_current_amd64.deb && \
-  echo "export CHROME_BIN=/opt/google/chrome/chrome" >> /usr/bin/bashwrapper
-
-# Keyman for Android
-RUN apt-get -q -y install gradle maven pandoc sdkmanager jq && \
-  sdkmanager platform-tools && \
-  yes | sdkmanager --licenses && \
-  chown -R build:build /opt/android-sdk/ && \
-  echo "export ANDROID_HOME=/opt/android-sdk" >> /usr/bin/bashwrapper && \
-  echo "export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64" >> /usr/bin/bashwrapper
-
-# Finish bashwrapper script and adjust permissions
-RUN echo "\${@:-bash}" >> /usr/bin/bashwrapper && \
-  chmod +x /usr/bin/bashwrapper && \
-  chown -R build:build $HOME
-
-# now, switch to build user
-USER build
-
-# Pre-install gradle. This will put files in ~/.gradle which will speed up builds.
-RUN mkdir -p $HOME/tmp/gradle/wrapper && \
-  # KMEA uses gradle-7.5.1-bin
-  curl --location --output $HOME/tmp/gradle/wrapper/gradle-wrapper.jar https://raw.githubusercontent.com/keymanapp/keyman/master/android/KMEA/gradle/wrapper/gradle-wrapper.jar && \
-  curl --location --output $HOME/tmp/gradle/wrapper/gradle-wrapper.properties https://raw.githubusercontent.com/keymanapp/keyman/master/android/KMEA/gradle/wrapper/gradle-wrapper.properties && \
-  curl --location --output $HOME/tmp/gradlew https://raw.githubusercontent.com/keymanapp/keyman/master/android/KMEA/gradlew && \
-  chmod +x $HOME/tmp/gradlew && \
-  $HOME/tmp/gradlew --quiet && \
-  # Some projects use gradle-7.5.1-all, so we pre-install that as well
-  curl --location --output $HOME/tmp/gradle/wrapper/gradle-wrapper.jar https://raw.githubusercontent.com/keymanapp/keyman/master/android/Samples/KMSample1/gradle/wrapper/gradle-wrapper.jar && \
-  curl --location --output $HOME/tmp/gradle/wrapper/gradle-wrapper.properties https://raw.githubusercontent.com/keymanapp/keyman/master/android/Samples/KMSample1/gradle/wrapper/gradle-wrapper.properties && \
-  curl --location --output $HOME/tmp/gradlew https://raw.githubusercontent.com/keymanapp/keyman/master/android/Samples/KMSample1/gradlew && \
-  chmod +x $HOME/tmp/gradlew && \
-  $HOME/tmp/gradlew --quiet && \
-  rm -rf $HOME/tmp
-
-ENTRYPOINT [ "/usr/bin/bashwrapper" ]
```

Comment:
Does this file need to be fully deleted

Reply:
yes, it's replaced by this PR. Splitting it in multiple Dockerfiles has the advantage that we notice if we have the documented dependencies wrong for a platform. Previously it was possible that e.g. the dependencies for Android were wrong in the documentation, but the build still worked locally because for Linux you already installed the missing dependencies. Now when you tried to set up a build agent that only builds Android things would fail.